app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
```

Comment:
You can simplify this logic by using `argv.index("-f")` to find the index of `-f` and assigning it to a `f_index` variable, then capture the filename by checking for the next element after this index:

```suggestion
if "-f" in argv:
    f_index = argv.index("-f")
    if f_index + 1 < len(argv):
        filename = argv[f_index + 1]
```